classloader-leak-prevention/classloader-leak-prevention-core/src/test/java/se/jiderhamn/classloader/leak/prevention/cleanup/JavaUtilLoggingLevelCleanUpTest.java

```diff
@@ -1,12 +1,19 @@
 package se.jiderhamn.classloader.leak.prevention.cleanup;
 
+import org.junit.Assume;
+
+import se.jiderhamn.classloader.leak.prevention.support.JavaVersion;
+
 /**
  * Test cases for {@link JavaUtilLoggingLevelCleanUp}
  * @author Mattias Jiderhamn
  */
 public class JavaUtilLoggingLevelCleanUpTest extends ClassLoaderPreMortemCleanUpTestBase<JavaUtilLoggingLevelCleanUp> {
   @Override
   protected void triggerLeak() throws Exception {
+      // Leak does not occur any more with JDK11+
+      Assume.assumeTrue(JavaVersion.IS_JAVA_10_OR_EARLIER);
```

Comment:
Why isn't `triggerLeakWithoutCleanup()` overridden in this case?

Reply:
Good question - I may have overlooked this one...